index.html

```diff
@@ -1,11 +1,153 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ru">
 <head>
     <meta charset="UTF-8">
+    <meta name="description" content="IT course">
+    <meta name="keywords" content="HTML,CSS,JS">
+    <meta name="author" content="Luda Zhuleha">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>First Project</title>
 </head>
 <body>
 <h1>Моя первая страница</h1>
 <p><a href="about/index.html">Об Авторе</a></p>
+<hr>
+<img src="img/coffee.jpg" alt="coffee-cup">
+<br>
+<ol>
+    <li>Знакомство с WEB
+        <ul>
+            <li>Краткая история WEB и что такое интернет.</li>
+            <li>Как устроен браузер и как он получает страницы по сети.</li>
+            <li>Что такое Git?</li>
+            <li>Создание первого репозитория.</li>
+            <li>О файле .gitignore.</li>
+        </ul>
+    </li>
+    <li>HTML-верстка: первые шаги
+        <ul>
+            <li>Настройка рабочей среды.</li>
+            <li>Краткая история HTM.</li>
+            <li>Какие бываю теги и что такое атрибут.</li>
+            <li>Теги для работы с текстом.</li>
+            <li>Создание собственной страницы.</li>
+        </ul>
+    </li>
+    <li>Погружение в HTML-разметку
+        <ul>
+            <li>Метатеги.</li>
+            <li>Мнемоники.</li>
+            <li>Тег для работы с изображениями.</li>
+            <li>Теги для работы со списками.</li>
+            <li>Теги для работы с таблицами.</li>
+            <li>Теги для работы с формами.</li>
+            <li>Div-блоки.</li>
+        </ul>
+    </li>
+</ol>
+<br>
+<table border="1" width="500">
+    <thead>
+    <tr>
+        <th>Tag</th>
+        <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>&lt; table &gt;</td>
+        <td>Defines a table</td>
+    </tr>
+    <tr>
+        <td>&lt; tr &gt;</td>
+        <td>Defines a row in a table</td>
+    </tr>
+    <tr>
+        <td>&lt; th &gt;</td>
+        <td rowspan="2">Defines a 'header cell'/ cell in a table</td>
+    </tr>
+    <tr>
+        <td>&lt; td &gt;</td>
+
+    </tr>
+    <tr>
+        <td colspan="2">Rarely used</td>
+    </tr>
+    <tr>
+        <td>&lt; caption &gt;</td>
+        <td>Defines a table caption</td>
+    </tr>
+    <tr>
+        <td>&lt; thead &gt;</td>
+        <td>Groups the header content in a table</td>
+    </tr>
+    <tr>
+        <td>&lt; tbody &gt;</td>
+        <td>Groups the body content in a table</td>
+    </tr>
+    <tr>
+        <td>&lt; tfoot &gt;</td>
+        <td>Groups the footer content in a table</td>
+    </tr>
+    </tbody>
+</table>
+<br>
+<fieldset>
```

Comment:
it's better to use `<fieldset>` and `<legend>` inside `<form>`

Reply:
Thanks! You are completely right. The form is corrected. Fieldset and legend are placed inside the form.